Governance.md

```diff
@@ -0,0 +1,323 @@
+broader ecosystem.
+
+## Governance
+
+The foundations of Project governance are:
+
+-   Openness and transparency
+-   Active contribution
+-   Institutional neutrality
+
+### Consensus-based decision-making by the community
+
+Normally, project decisions will be made by consensus of all interested
+Contributors. The primary goal of this approach is to ensure that the people who
+are most affected by and/or involved in any given change can contribute their
+knowledge in the confidence that their perspective will be acknowledged.
+Thoughtful review from a broad community is the best mechanism we know of for
+creating high-quality software.
+
+In this context, consensus does not require:
+
+-   that we solicit everybody's opinion on every change
+-   that we ever hold a vote on anything
+-   that everybody is happy or agrees with every decision
+
+Consensus means that we entrust everyone with the right to veto any change if
+they feel it necessary.
+
+To achieve consensus, we use a combination of best judgment and best efforts:
+
+-   A simple and uncontroversial bug fix posted on GitHub and reviewed by a core
+    developer is probably fine.
+-   Substantive or controversial API changes may involve one or more GitHub
+    issues, Pull Requests, or Discussions that are open for longer periods of
+    time to provide greater opportunity for feedback from the Community and may
+    specifically solicit feedback from targeted members of the Community.
+
+We expect that anyone who cares enough about The Project will raise their
+concerns or express their formal veto within one of these channels within a
+reasonably short period of time (up to a few weeks). If no major unresolved
+concerns exist after a period of time appropriate for the scope of the change,
+we will assume that consensus has been reached. If we find that a given change
+is more controversial than expected or a critical input is delayed because
+someone was unavailable, we can back out or further modify those changes based
+on the additional feedback.
+
+If necessary, a formal veto should consist of:
+
+-   an unambiguous statement that a veto is being invoked
+-   an explanation of why it is being invoked
+-   a description of what conditions (if any) would convince the vetoer to
+    withdraw their veto
+
+We expect that most people will take this responsibility seriously and only
+invoke their veto when they judge that a serious problem is being ignored and
+that their veto is necessary to protect The Project. We expect such vetoes to be
+rare in practice because Contributors are motivated from the start to find some
+solution that everyone can live with.
+
+If all proposals for resolving some issue are vetoed, then the status quo wins
+by default. In the worst case, if a Contributor is genuinely misusing their veto
+in an obstructive fashion to the detriment of the project, then they can be
+ejected from the project by consensus of the Steering Council – see below.
```

Comment:
```suggestion
ejected from the project by consensus of the Steering Council in the process described in the **Council Membership** section.
```

Reply:
I think the "see below" was referring more to the Steering Council section than the process by which consensus is determined.  Since this occurs in the same section where we describe consensus, I think we can omit a reference back to it.  Which then leaves us with 

```Markdown
then they can be ejected from the project by consensus of the Steering Council.
```

(dropping further qualification)

Reply:
Sure, that works!